fluentd/standalone-image/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG IMAGE_VERSION=v1.14.0-debian-1.0
```

Comment:
just to make sure, the image we usually use, the daemonset image didn't work here? it couldnt run with it ?

Reply:
correct, that is why i took the non daemonset image